packages/web3-eth-accounts/src/index.js

```diff
@@ -25,18 +25,17 @@
 var _ = require('underscore');
 var core = require('web3-core');
 var Method = require('web3-core-method');
-var Promise = require('any-promise');
```

Comment:
`any-promise` goes back in here?

Reply:
I've checked that closer and the ``any-promise`` polyfill dependency is actually not required because of the usage of the ``@babel/plugin-transform-runtime`` plugin. This plugin does polyfill such built-ins with ``corejs`` or ``@babel/polyfill`` as described [here](https://babeljs.io/docs/en/babel-plugin-transform-runtime).

Reply:
Ok I'm defeated by this ha ha! but left some questions about transpilation in the review.